src/Transformers/JsonApi/requestSerializer.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @param {Object} data
+ * @param {Object} headers
+ */
+export default function requestSerializer(data, headers) {
+
```

Comment:
Het kan geen kwaad hier al `return data;` toe te voegen. Dat is zeg maar de identity serializer.